# Executes joints buffer (synchronized)
---
bool success
string msg
